config/datocms/migrations/1741855116_addGroupingBlock.ts

```diff
@@ -0,0 +1,224 @@
+import { Client } from '@datocms/cli/lib/cma-client-node';
+
+export default async function (client: Client) {
+  console.log('Create new models/block models');
+
+  console.log(
+    'Create block model "\uD83D\uDDC3\uFE0F Grouping Block" (`grouping_block`)',
+  );
+  await client.itemTypes.create(
+    {
+      id: 'TBuD6qQOSDy6i9dM3T_XEA',
+      name: '\uD83D\uDDC3\uFE0F Grouping Block',
+      api_key: 'grouping_block',
+      modular_block: true,
+      draft_saving_active: false,
+      inverse_relationships_enabled: false,
+    },
+    {
+      skip_menu_item_creation: true,
+      schema_menu_item_id: 'XZDLSGZcSvGjkpzdfymBrg',
+    },
+  );
+
+  console.log(
+    'Create block model "\uD83D\uDDC2\uFE0F Grouping Item Block" (`grouping_item_block`)',
```

Comment:
Should we maybe name it `GroupingItem` instead of `GroupingItemBlock`? It should only be used inside the `GroupingBlock`, and should not be in `AnyBlock`.